llvm/test/DebugInfo/fixed-point.ll

```diff
@@ -0,0 +1,26 @@
+;; This fixes https://github.com/llvm/llvm-project/issues/81555
+;; Note this isn't correct dwarf but we just need to assert this doesn't crash.
```

Comment:
If this isn't correct, should we instead reject it in the IR Verifier?

Reply:
Updated. Assuming I'm using the verify correctly, it looks like this dwarf actually is valid.